mono/metadata/image.c

```diff
@@ -1920,8 +1920,8 @@ mono_image_open_full (const char *fname, MonoImageOpenStatus *status, gboolean r
 	return mono_image_open_a_lot (fname, status, refonly, FALSE);
 }
 
-MonoImage *
-mono_image_open_a_lot (const char *fname, MonoImageOpenStatus *status, gboolean refonly, gboolean load_from_context)
+static MonoImage *
+mono_image_open_a_lot_parameterized (const char *fname, MonoImageOpenStatus *status, gboolean refonly, gboolean load_from_context, gboolean *problematic)
```

Comment:
nit: 😢

Reply:
I feel like making every external caller see the options to the core workhorse function is a bit worse than adding another internal/external split. Future options added can just be added to the internal one.

I agree though. 😢